### ISSUE TITLE
Request to add StyleTTS 2 model

### ISSUE THREAD

**Issue by athyfr** [mkiol/dsnote#183]
Looking around, I've discovered the StyleTTS 2 model.

It seems to be of a much higher quality than other Voice Cloning TTS models, so I think it would be nice to support.

From the statistics mentioned in [it's page,](https://styletts2.github.io/) it seems to be a little more optimised than YourTTS, which is already supported by SpeechNote.

Alike to what SpeechNote does, it seems to work well when generated on a sentence-by-sentence basis. However, the quality degrades on smaller segments of text.

There is a sweet spot to maximize quality.

You can find the source [here](https://github.com/yl4579/StyleTTS2).

Thanks for the awesome program!

**Comment by mkiol** [comment]
Thanks for the recommendation. Adding to my TO-DO list. 

BTW, Have you tested the Coqui XTTS model? In my opinion it is great, much better than YourTTS. Especially in version 2.0.2.

**Comment by athyfr** [comment]
> Thanks for the recommendation. Adding to my TO-DO list.
> 
> BTW, Have you tested the Coqui XTTS model? In my opinion it is great, much better than YourTTS. Especially in version 2.0.2.

Yeah, I've tried it. It is certainly much better! I think Style TTS is higher quality than that though, the demos show it as pretty good.
